docs/design/disaster-recovery-optimization.md

```diff
@@ -0,0 +1,52 @@
+# FEATURE: Regional Disaster Recovery Optimization
+
+## Summary
+
+Implement use of `bluestore-rdr` as the OSD backend store for customers using ODF with Regional Disaster Recovery (RDR).
+
+## Motivation
+
+- Ceph uses BlueStore as a special-purpose storage back end designed specifically for managing data on disk for Ceph OSD workloads. Ceph team is working on improving this backend storage that will help in significantly improving the performance, especially in disaster recovery scenarios. This new OSD backend is known as bluestore-rdr. In the future, other backends will also need to be supported, such as seastore.
+
+## Proposal
+
+- Enable the optimized OSD backend store for RDR clusters.
+- Enable this optimization for both new (Greenfield) and existing (brownfield) customers.
+
+### Greenfield
+- If `ocs.openshift.io/optimizeDisasterRecovery: true` annotation is available in the `StorageCluster` then configure OSD store type in the `cephCluster` as:
```

Comment:
@umangachapagain are you ok with the annotation?

Reply:
I'd prefer avoiding "optimize" and keeping it generic like "clusterIsDisasterRecoveryReady" or "clusterIsDisasterRecoveryTarget" etc since we might not always be doing optimization operations. But, I don't mind this as well.

Reply:
"enabled" would be misleading, since user wants to enable it but might not have done that already.

Reply:
got it.